index.html

```diff
@@ -0,0 +1,164 @@
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
+                </ul>
+            </div>
+
+            <nav class="navbar">
+                <a href="index.html" class="nav-branding"><img src="images/dark logo.png" alt="logo"></a>
+
+                <ul class="nav-menu">
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link orange-text">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#program" class="nav-link">Program</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#featured" class="nav-link">Featured</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#sponsor" class="nav-link">Sponsor</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="/" class="nav-link border-text">Community</a>
+                    </li>
+                </ul>
+
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+            </nav>
+        </header>
+        <!--HERO-->
+        <section class="hero">
+            <p class="quote">
+                “Welcome to the javascript society summit”
+            </p>
+            <h1 class="first-heading">
+                Javascript society<br>
+                Summit<br>
+                2023
+            </h1>
+            <div class="first-para">
+                <p>
+                    An Javascript coders community that believes in the value of openness, sharing, creativity and tech
+                    empowerment. Creating a positive change with people all over the world. We are hosting Africa
+                    biggest open source conference this November, in Nairobi, Kenya.
+                </p>
+            </div>
+            <h2 class="second-heading">
+                2023.07.3(MON) ~ 7(FRI)
+            </h2>
+            <p class="second-para">
+                @ UON Towers, Nairobi, Kenya and more.
+            </p>
+        </section>
+    </div>
+    <!--MAIN PROGRAM-->
+    <section id="program" class="main-program">
+        <div class="main-heading">
+            <h2>Main Program</h2>
+            <hr class="main-line">
+        </div>
+        <div class="program-cards">
+            <div class="cards">
+                <img src="images/icon1.png" alt="icon">
+                <h3>Keynote</h3>
+                <p>Renowned experts in the field deliver inspiring talks on the latest trends,
+                    technologies, and best practices in Javascript society.
+                </p>
+            </div>
```

Comment:
I think it wil be better to work on the colors suggestions given by the previous reviewer, as your background design seems to be covering the design of some of your texts and this will lead to accessibility issues. Currently, your `data and location` in the homepage is the same color with your background and it's not visible. It is always advisable to use a dark background and a light text or a light background and a light text and ensure you follow color guidelines for this project 😄 

Lastly, your Heading `Main program`  is now colliding with your venue and time. please fix this as well 👍 

Your design :

![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/825828a4-51a2-4654-b2e0-feedf411dbfb)

Design given :
![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/36f61d86-f684-4c2f-b443-f7c64514fe04)